locales/app.yml

```diff
@@ -56,6 +56,14 @@ _version: 2
   zh_CN: "正在拉取"
   zh_TW: "正在拉取"
   de: "Abrufen"
+"Fetching":
+  en: "Fetching"
+  lt: "Gaunama"
+  es: "Obteniendo"
+  fr: "Récupération"
+  zh_CN: "正在获取"
+  zh_TW: "正在獲取"
+  de: "Abrufen"
```

Comment:
Some of these overlap (fr, zh*, de). Is there some translated Git documentation that you can reference?